### ISSUE TITLE
Some monsters are incorrectly identified as a summon in character deaths.

### ISSUE THREAD

**Issue by Auscent** [TibiaData/tibiadata-api-go#89]
This happens to normal monsters who have ' of ' in their name and prefixed with 'a' or 'an' in the death message which causes them to match against summonRegex. A few examples would be sight of surrender, spark of destruction and the X of the cult family. A simple solution at a glance seems to be moving the get summon block inside TibiaDataParseKiller into the already existing if player conditional branch, and I don't believe this would affect deaths by summons of monsters since as far as I know they're always listed separately from their owner.

**Comment by tobiasehlert** [comment]
Thanks for reporting this @Auscent!
Do you have examples of characters, where you are experiencing this issue?

**Comment by Auscent** [comment]
Zetern Thoth

I noticed the issue with another character who had died by a sight of surrender a month ago, that death has now expired from the list. I put a fresh death on that character by a novice of the cult though to verify the issue which is interpreted by TibiaData as "the cult" for killer and "a novice" for summon.

**Comment by tobiasehlert** [comment]
Thanks @Auscent!
We'll look into this :)

---

URL:
- https://www.tibia.com/community/?name=Zetern+Thoth
- https://api.tibiadata.com/v3/character/Zetern+Thoth

Actual result:
```json
{
	"characters": {
		"deaths": [{
			"time": "2022-03-03T07:02:26Z",
			"level": 19,
			"killers": [{
				"name": "the cult",
				"player": false,
				"traded": false,
				"summon": "a novice"
			}],
			"assists": [],
			"reason": "Died at Level 19 by a novice of the cult."
		}]
```

Expected result:
```json
{
	"characters": {
		"deaths": [{
			"time": "2022-03-03T07:02:26Z",
			"level": 19,
			"killers": [{
				"name": "a novice of the cult",
				"player": false,
				"traded": false,
				"summon": ""
			}],
			"assists": [],
			"reason": "Died at Level 19 by a novice of the cult."
		}]
```

**Comment by tobiasehlert** [comment]
Thanks for reporting, this should be resolved with PR #116 👍
